tests/BasicSearchTests.FunctionalTests.Core/Constants.cs

```diff
@@ -6,7 +6,8 @@ namespace BasicSearchTests.FunctionalTests.Core
     public static class Constants
     {
         // Predefined Texts
-        public const string ExistentSearchString = "Newtonsoft.Json";
+        public const string TestPackageId = "BaseTestPackage";
```

Comment:
I tested this across all environments and ensured the package existed and its metadata matched what the functional test expects.